releases/2022-11-25-releasenotes.md

```diff
@@ -0,0 +1,154 @@
+---
+draft: false
+title: "Handle errors the GraphQL way and more results to your query"
+description: "Vendia Share updated the response status code and allowing up to 1000 query results"
+date: '2022-11-25'
+authors:
+  - Li Dai
+tags:
+- Improvements
+- Developer Experience
+- GraphQL
+---
+
+Vendia Engineers worked tirelessly to improve our product and ensure that we follow and set best practices for industrial standards. Here are the newest updates!
+
+# Error handling the way it should with [GraphQL](https://graphql.org/). 
+
+Errors are not scary as long as we handle them well. In the past Vendia's end point would return 4XX and 5XX status codes for any http request failures. But that has changed as we decided to follow GraphQL's [recommended approach](https://graphql.github.io/graphql-over-http/draft/#sec-Field-errors-encountered-during-execution) over http calls.
+
+In short, it means the end point will always return 2XX. But when error happens, error messages will be included in the response. 
```

Comment:
Is that true? I was under the impression there were scenarios in which we return both 200 and 400 error codes.

Reply:
Per this doc, it looks like there are instances of 4xx error: https://www.notion.so/vendia/GraphQL-Improving-our-error-handling-efbabbcef21a4758a93dff16e120d287

Reply:
This doc seems to be outdated. The [2022-11-04 release](https://github.com/vendia/vendia/blob/master/releasenotes/releases/RELEASE_NOTES_prod-release.2022-11-04.e4105a1.rst) had updated all the RESTful status code standards to 200 and have error field within the response body.